examples/search_by_date.py

```diff
@@ -0,0 +1,59 @@
+from python_http_client import Client
+import os
+import json
+
+all_repos = [
+    'sendgrid-nodejs',
+    'sendgrid-csharp',
+    'sendgrid-php',
+    'sendgrid-python',
+    'sendgrid-java',
+    'sendgrid-go',
+    'sendgrid-ruby',
+    'smtpapi-nodejs',
+    'smtpapi-go',
+    'smtpapi-python',
+    'smtpapi-php',
+    'smtpapi-csharp',
+    'smtpapi-java',
+    'smtpapi-ruby',
+    'sendgrid-oai',
+    'open-source-library-data-collector',
+    'python-http-client',
+    'php-http-client',
+    'csharp-http-client',
+    'java-http-client',
+    'ruby-http-client',
+    'rest',
+    'nodejs-http-client',
+    'dx-automator'
+]
+
+start_date = "2019-01-01"
+end_date = "2019-01-31"
+
+def get_items(repo, item_type):
+    client = Client(host="http://{}".format(os.environ.get('DX_IP')))
+    query_params = {
+        "repo":repo,
+        "item_type":item_type,
+        "states[]":['OPEN'],
+        "start_creation_date": start_date,
+        "end_creation_date": end_date,
+        "limit[]":['first', '100']
```

Comment:
nit: Limiting the results doesn't make sense here.

Reply:
Removing this line gives an error and I think limit needs to be specified in GraphQL? @thinkingserious